### ISSUE TITLE
Additional folders not syncing or appearing in snapshots

### ISSUE THREAD

**Issue by o0ryan0o** [Jumoo/uSync.Complete.Issues#28]
**Describe the bug**
Additional folders not syncing or appearing in snapshots.  I am trying to sync two additional folders in my VS project named /img and /webfonts.

**Expected behavior**
I've added the above paths to the snapshot settings and the uSync.Publish.config.  When I publish with uSync, changes in the /img and /webfonts folder are not making it to the test server.  When I create a snapshot, these directories are not appearing in the snapshot zip file as well.

**Desktop (please complete the following information):**
 - OS: Windows 10 & Server 2012 R2
 - Browser: Chrome
 - Umbraco: 8.6.1
- uSync Complete Trial: 8.6 installed via Nuget

I know I could publish these files via Visual Studio, but I was hoping to use uSync.  That way we can move changes around Dev to Test, and eventually Live.  Any help is greatly appreciated.  Thank you!

**Comment by KevinJump** [comment]
Hi,

We need to update documentation but the snapshots folders value actually comes from the usync8.config file

adding this under the `<BackOffice>` section (and restarting the site) should do it.

```
    <Snapshots>
      <folders>views,css,scripts,img,webfonts</folders>
    </Snapshots>
```

**Comment by o0ryan0o** [comment]
Thank you for the quick reply!  I can see that line gets updated in the usync8.config when I save in the backoffice.  I actually had 3 lines that said:
<folders>views,css,scripts,img,webfonts</folders>
<folders>views,css,scripts,img,webfonts</folders>
<folders>views,css,scripts,img,webfonts</folders>

I removed the extras previously.  When working in Visual Studio, I assumed a complete project rebuild would be good enough as restart.  I've also tried stopping the IIS Express site and rerunning the site.  I still can't get those directories to appear.  Should I maybe completely close VS and reboot my machine :)  Thanks again!
